cssdb.json

```
@@ -526,6 +526,15 @@
       }
     ]
   },
+  {
+    "id": "read-only-write",
+    "title": "`:read-only` and `:read-write` selectors",
+    "description": "Pseudo-classes to match elements which are considered user-alterable",
+    "specification": "https://drafts.csswg.org/selectors-4/#rw-pseudos",
+    "stage": 1,
```

Could this be changed to Stage 2? This has advanced from csswg.org to w3.org.

```suggestion
    "stage": 0,
```

---

Agreed. It is working draft